s3fs/tests/test_s3fs.py

```diff
@@ -1871,3 +1890,32 @@ def test_get_file_info_with_selector(s3):
                 raise ValueError("unexpected path {}".format(info["name"]))
     finally:
         fs.rm(base_dir, recursive=True)
+
+
+@pytest.mark.xfail(
+    skip=version.parse(moto.__version__) > version.parse("1.3.16"),
```

Comment:
Should be `condition=`, and I think this is the wrong way around: this test is expected to fail is the current version is 1.3.16 or less. It should pass for the next moto.

Reply:
Good catch! Thanks, I corrected it.